README.rst

```diff
@@ -71,6 +74,8 @@ if you do so, then we'll be able to include it here and spread the word about it
 Installation
 -------------
 
+Follow our `getting started guide <http://coala.readthedocs.io/en/latest/Users/Tutorials/Tutorial.html>`_ or these quick start instructions
+
```

Comment:
i dont like this addition, I think the Installation section is clear as it is :/